src/main/webapp/js/userMapData.json

```diff
@@ -752,7 +805,9 @@
 ["University of Saint Mary","USA"],
 ["University of San Diego","USA"],
 ["University of Scranton","USA"],
+["University of South Carolina","USA"],
 ["University of South Carolina Aiken","USA"],
```

Comment:
These are different - University of South Carolina is in Columbia, and University of South Carolina Aiken is in Aiken.